utils/build-script-impl

```diff
@@ -2695,6 +2712,12 @@ for host in "${ALL_HOSTS[@]}"; do
                 echo "Cleaning the Foundation build directory"
                 call rm -rf "${build_dir}"
 
+                # Set the PKG_CONFIG_PATH so that core-foundation can find the libraries and
+                # header files
+                LIBICU_BUILD_DIR="$(build_directory ${host} libicu)"
+                export PKG_CONFIG_PATH="${LIBICU_BUILD_DIR}/config:${PKG_CONFIG_PATH}"
```

Comment:
Why were the CMake options removed?

Reply:
Which ones? Maybe this comment was on the wrong line?  There was a set of options being appended to `swift_cmake_options` during the setup of ICU that I moved directly into the configuration for swift.